InstrumentationEngine.sln

```diff
@@ -92,535 +92,669 @@ Global
+		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Debug|ARM64.ActiveCfg = Debug|ARM64
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Debug|x64.ActiveCfg = Debug|x64
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Debug|x64.Build.0 = Debug|x64
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Debug|x86.ActiveCfg = Debug|Win32
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Debug|x86.Build.0 = Debug|Win32
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Release|Any CPU.ActiveCfg = Release|Win32
+		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Release|ARM64.ActiveCfg = Release|ARM64
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Release|x64.ActiveCfg = Release|x64
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Release|x64.Build.0 = Release|x64
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Release|x86.ActiveCfg = Release|Win32
 		{1C6954D1-1A98-4149-8D2C-B0D49D03A832}.Release|x86.Build.0 = Release|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Win32
+		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|ARM64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Managed|x64.ActiveCfg = Release|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Managed|x86.ActiveCfg = Release|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Native|Any CPU.ActiveCfg = Release|Win32
+		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Native|ARM64.ActiveCfg = Debug|ARM64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Native|x64.ActiveCfg = Release|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Native|x64.Build.0 = Release|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Native|x86.ActiveCfg = Release|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.CodeAnalysis.Native|x86.Build.0 = Release|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Debug|Any CPU.ActiveCfg = Debug|Win32
+		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Debug|ARM64.ActiveCfg = Debug|ARM64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Debug|x64.ActiveCfg = Debug|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Debug|x64.Build.0 = Debug|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Debug|x86.ActiveCfg = Debug|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Debug|x86.Build.0 = Debug|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Release|Any CPU.ActiveCfg = Release|Win32
+		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Release|ARM64.ActiveCfg = Release|ARM64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Release|x64.ActiveCfg = Release|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Release|x64.Build.0 = Release|x64
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Release|x86.ActiveCfg = Release|Win32
 		{DAB53F24-D55A-4759-B100-5B7FEFBF6D08}.Release|x86.Build.0 = Release|Win32
 		{8F446E35-5683-4ABF-8F42-1C8EB5797A5B}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Any CPU
 		{8F446E35-5683-4ABF-8F42-1C8EB5797A5B}.CodeAnalysis.Managed|Any CPU.Build.0 = Release|Any CPU
+		{8F446E35-5683-4ABF-8F42-1C8EB5797A5B}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|Any CPU
```

Comment:
>Any CPU [](http://example.com/codeflow?start=86&length=7)

VS Configuration Manager does not like these projects having ARM64 and keeps reverting them so I'll keep these as `Any CPU`